components/List/tests/snapshot/MDCSelfSizingStereoCellSnapshotTests.m

```diff
@@ -78,4 +136,30 @@ - (void)testCellWithTitleAndDetailAndImage {
   [self generateSnapshotAndVerifyForView:self.cell];
 }
 
+- (void)testCellWithDynamicTypeForContentSizeCategoryExtraSmallEnabledForTitleAndDetail {
+  if (@available(iOS 10.0, *)) {
+    // When
+    self.cell.titleLabel.text = @"Title";
+    self.cell.detailLabel.text = @"Detail";
+    self.cell.mdc_adjustsFontForContentSizeCategory = YES;
```

Comment:
It looks like you forgot to set a font with attached scaling curves here. Please be sure to add them. If you are intended to test the current/legacy behavior, then I suspect you'll need to change the font to something that's not the default, then verify that the `adjustsFontForContentSizeCategory` replaces it with a Material Typography font.

Reply:
Addressed in 2a572487eb4ec3015fa52debd41cd3387e10d968. Added fonts with attached scaling curves.